milabowl-astro/src/pages/index.astro

```diff
@@ -1,75 +1,96 @@
 import game_state from "../../src/game_state/game_state.json";
 import type { MilaResultsDTO } from "../../src/game_state/gameState";
-import Link from '../components/core/Link.astro';
-import PositionDelta from '../components/core/PositionDelta.astro';
+import Link from "../components/core/Link.astro";
+import PositionDelta from "../components/core/PositionDelta.astro";
+
 const milaResults: MilaResultsDTO = game_state;
 ---
+
 <Layout title="Milabowl - Dashboard">
-	<main>
-		<div class="flex flex-col lg:grid lg:grid-cols-12 gap-4">	
-			<div class="lg:col-span-5">
-				<Card title="Standings">
-					<div slot="secondary">
-						<Link href="/standings">See More</Link>
-					</div>
-					<Table>
-						<tr slot="header">
-							<th>Rank</th>
-							<th>Team</th>
-							<th class="right-align">GW</th>
-							<th class="right-align hidden md:table-cell">Avg</th>
-							<th class="right-align table-cell md:hidden">Total</th>
-							<th class="right-align hidden md:table-cell">Total Score</th>
-						</tr>							
+  <main>
+    <div class="flex flex-col lg:grid lg:grid-cols-12 gap-4">
+      <div class="lg:col-span-5">
+        <Card title="Standings">
+          <div slot="secondary">
+            <Link href="/standings">See More</Link>
+          </div>
+          <Table>
+            <tr slot="header">
+              <th>Rank</th>
+              <th>Team</th>
+              <th class="right-align">GW</th>
+              <th class="right-align hidden md:table-cell">Avg</th>
+              <th class="right-align table-cell md:hidden">Total</th>
+              <th class="right-align hidden md:table-cell">Total Score</th>
+            </tr>
+
+            {
+              milaResults.resultsByWeek[
+                milaResults.resultsByWeek.length - 1
+              ].results.map((r, i) => {
+                const deltaPosition =
+                  r.milaRankLastWeek === null ||
+                  r.milaRankLastWeek === undefined
+                    ? 0
+                    : r.milaRankLastWeek - r.milaRank;
+
+                const isLastPlace =
+                  milaResults.resultsByWeek[
+                    milaResults.resultsByWeek.length - 1
+                  ].results.length -
+                    1 ===
+                  i;
+                return (
+                  <tr class="border-1 border-slate-200 dark:border-slate-700">
+                    <td class="pr-0">
+                      <div class="flex space-x-2 flex-row items-center justify-items-center">
+                        <div>
+                          {isLastPlace && <span class="text-md">💩</span>}
+                          {!isLastPlace && `${i + 1}.`}
+                        </div>
+                        <PositionDelta delta={deltaPosition} />
+                      </div>
+                    </td>
+                    <td>
+                      <Avatar
+                        team_name={r.teamName}
+                        avatar_size="20"
```

Comment:
Ville vurdert en litt større avatar faktisk (men må funke på mobil). Her kan du bruke media queries for å ha ulike størelser basert på skjermstørrelse. Dette er innebygd i tailwind: https://tailwindcss.com/docs/hover-focus-and-other-states#responsive-breakpoints

Reply:
Takk for input. Da endret jeg til statiske størrelser (altså ikke via size prop'en), og satt litt ulike for små og store skjermer. Ser ut som det funker greit for mobil :)